cmd/fleetctl/setup.go

```diff
@@ -49,6 +49,12 @@ func setupCommand() cli.Command {
 				return err
 			}
 
+			if flEmail == "" {
+				return fmt.Errorf("Email of the admin user to create must be provided")
```

Comment:
errors.Errorf here too :)

Reply:
Rest of `fleetctl` uses `fmt.Errorf`, and there is mixed usage in `server/` and `fleet`.

Project wide s/fmt.Errorf/errors.Errorf` ?

Reply:
Yup, we're just progressively refactoring for now but we can totally put together a PR for a complete refactor.